modules/react/action-bar/lib/ActionBarList.tsx

```diff
@@ -56,21 +59,9 @@ export const ActionBarList = createSubcomponent('div')({
   elemPropsHook: useActionBarList,
 })<ActionBarListProps>(({children, overflowButton, ...elemProps}, Element, model) => {
   return (
-    <ResponsiveList
-      as={Element}
-      gap="s"
-      depth={1}
-      background={commonColors.background}
-      borderTop={`solid 1px ${colors.soap400}`}
-      padding={`${space.s} ${space.xl} `}
-      position="fixed"
-      bottom={0}
-      left={0}
-      right={0}
-      {...elemProps}
-    >
+    <Flex as={Element} {...mergeStyles(elemProps, responsiveListStencil())}>
```

Comment:
We've had the habit of removing this lower elements, so you can probably clean this up to be `<Element {...mergeStyles(elemProps, responsiveListStencil())}>`

in this case `Element` is `div` cause that's what we're passing to `createSubcomponent`

Reply:
You'll have to add `display: 'flex'` to the base styles in the stencil